aws/resource_aws_transfer_server.go

```diff
@@ -52,8 +54,9 @@ func resourceAwsTransferServer() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"vpc_endpoint_id": {
-							Type:     schema.TypeString,
-							Required: true,
+							Type:          schema.TypeString,
+							Optional:      true,
+							ConflictsWith: []string{"endpoint_details.0.address_allocation_ids", "endpoint_details.0.subnet_ids", "endpoint_details.0.vpc_id"},
 							ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
I suggest removing this `ValidateFunc`, we don't normally check for resource ID formatting. Thanks.

Reply:
done